docs/OV_Runtime_UG/automatic_batching.md

```diff
@@ -90,44 +100,45 @@ For example, the application processes only 4 video streams, so there is no need
-| "AUTO_BATCH_TIMEOUT" | timeout value, in ms | 1000 |  you can reduce the timeout value (to avoid performance penalty when the data arrives too non-evenly) e.g. pass the "100", or in contrast make it large enough e.g. to accommodate inputs preparation (e.g. when it is serial process)     |
+| `AUTO_BATCH_DEVICE` | The name of the device to apply Automatic batching,  with the optional batch size value in brackets. | N/A | `BATCH:GPU` triggers the automatic batch size selection. `BATCH:GPU(4)` directly specifies the batch size.     |
+| `AUTO_BATCH_TIMEOUT` | The timeout value, in ms. | 1000 |  You can reduce the timeout value to avoid performance penalty when the data arrives too unevenly). For example, set it to "100", or the contrary, i.e., make it large enough to accommodate input preparation (e.g. when it is a serial process).     |
 
-### Testing Automatic Batching Performance with the Benchmark_App
-The `benchmark_app`, that exists in both  [C++](../../samples/cpp/benchmark_app/README.md) and [Python](../../tools/benchmark_tool/README.md) versions, is the best way to evaluate the performance of the Automatic Batching:
- -  The most straighforward way is performance hints:
+## Testing Performance with Benchmark_app
+The `benchmark_app` sample, that has both [C++](../../samples/cpp/benchmark_app/README.md) and [Python](../../tools/benchmark_tool/README.md) versions, is the best way to evaluate the performance of Automatic Batching:
+ -  The most straightforward way is using the performance hints:
     - benchmark_app **-hint tput** -d GPU -m 'path to your favorite model'
- -  Overriding the strict rules of implicit reshaping by the batch dimension via the explicit device notion:
+ -  You can also use the "explicit" device notion to override the strict rules of the implicit reshaping by the batch dimension:
     - benchmark_app **-hint none -d BATCH:GPU** -m 'path to your favorite model'
- -  Finally, overriding the automatically-deduced batch size as well:
+ -  or override the automatically deduced batch size as well:
     - $benchmark_app -hint none -d **BATCH:GPU(16)** -m 'path to your favorite model'
-    - notice that some shell versions (e.g. `bash`) may require adding quotes around complex device names, i.e. -d "BATCH:GPU(16)"
+    - This example also applies to CPU or any other device that generally supports batch execution.
+    - Note that some shell versions (e.g. `bash`) may require adding quotes around complex device names, i.e. `-d "BATCH:GPU(16)"` in this example.
 
-The last example is also applicable to the CPU or any other device that generally supports the batched execution.  
+Note that Benchmark_app performs a warm-up run of a _single_ request. As Auto-Batching requires significantly more requests to execute in batch, this warm-up run hits the default timeout value (1000 ms), as reported in the following example:
```

Comment:
In my comment earlier, there is some code and several sentences. We need to add them back, otherwise, the content is not complete. I could not put them in one comment for some reason.